chapters/en/slides/module8/module8_00.md

```diff
@@ -10,22 +10,20 @@ type: slides
 
 By the end of this module, you will be able to
 
-- Explain the Jupyter Book ecosystem
-- Install Jupyter Book
-- Build your book
-- Create your own content files
-- Publish your book online
+1. Explain the Jupyter Book ecosystem.
+2. Install Jupyter Book.
```

Comment:
Again, is this not already installed at this point?

Reply:
Yup it is, I removed this LO and will clarify in the slide that it is repetition when we talk about installation.